inference-engine/tests_deprecated/unit/inference_engine_tests/util_const_infer_test.cpp

```diff
@@ -587,7 +587,7 @@ TEST_F(RemoveLayerTests, canGetConstDataForUnknownImpl) {
     ASSERT_EQ(getData("data9")->getTensorDesc().getDims(), refShape);
 }
 
-TEST_F(RemoveLayerTests, canFoldConstSubgraphs) {
+TEST_F(RemoveLayerTests, DISABLED_canFoldConstSubgraphs) {
```

Comment:
Can we remove these tests? Looks like these tests won't be enabled anymore.

Reply:
agree, removed